docs/realtime-chat.md

```diff
@@ -0,0 +1,249 @@
+# Realtime Chat
+
+Realtime chat makes it easy to power a live community.
+
+Users can chat with eachother 1-on-1, as part of a group, and in chat rooms. Messages can contain images, links, and other content. These messages are delivered immediately to clients if the recipients are online and stored in message history so offline users can catch up when they connect.
+
+Every message which flows through the realtime chat engine belongs to a topic which is used internally to identify which users should receive the messages. Users explicitly join and leave topics when they connect. This makes it easy to selectively listen for messages which they care about or decide to "mute" certain topics when they're busy. Users can also join multiple topics at once to chat simultaneously in multiple groups or chat rooms.
+
+There are 3 types of topic:
+
+1. A chat room is great for public chat. Any user can join and participate without need for permission. These rooms can scale to millions of users all in simultaneous communication. This is perfect for live participation apps or games with live events or tournaments.
+
+2. A group chat is private to only users part of a [group](groups-clans.md). Each user must be a member of the group and no other users can participate. You can use group chat with team-based gameplay or collaboration.
+
+3. Direct chat is private between two users. Each user will receive a [notification](in-app-notifications.md) when they've been invited to chat. Both users must join for messages to be exchanged which prevents spam from bad users.
```

Comment:
"Both users must join for messages to be exchanged " -> "Both users must agree to join this topic for messages to be exchanged"